extensions/wikia/Email/i18n/TwitchDisconnect.i18n.php

```diff
@@ -0,0 +1,94 @@
+<?php
```

Comment:
I think you should create `crowdin-twitchdisconnect.conf` file (based on configs we have for FB and Google).